mmv1/third_party/terraform/website/docs/r/logging_project_sink.html.markdown

```diff
@@ -89,6 +89,47 @@ resource "google_project_iam_binding" "gcs-bucket-writer" {
 }
 ```
 
+The following example creates a sink that are configured with user-managed service accounts, by specifying
+the `custom_writer_identity` field.
+
+Note that you can only create a sink that uses a user-managed service account when the sink destination
+is a log bucket.
+
+```hcl
+resource "google_service_account" "custom-sa" {
+  project      = "other-project-id"
+  account_id   = "gce-log-bucket-sink"
+  display_name = "gce-log-bucket-sink"
+}
+
+# Create a sink that uses user-managed service account
+resource "google_logging_project_sink" "my-sink" {
+  name = "other-project-log-bucket-sink"
+
+  # Can export to log bucket in another project
+  destination = "logging.googleapis.com/projects/other-project-id/locations/global/buckets/gce-logs"
+
+  # Log all WARN or higher severity messages relating to instances
+  filter = "resource.type = gce_instance AND severity >= WARNING"
+
+  # unique_writer_identity has to be true in order to use user-managed service account
```

Comment:
You can probably leave this out due to the default (and upcoming API behavior change)

Reply:
Done.